admin-dev/themes/default/js/bundle/product/product-combinations.js

```diff
@@ -217,6 +217,13 @@ var combinations = (function() {
           number.text(countSelectedProducts() + '/' + allProductCombination);
         });
 
+        $(function() {
+            $('#combination_form_' + contentElem.attr('data')).find("img").each(function() {
+                title = $(this).attr('src').split('/').reverse()[0];
```

Comment:
You can also `.pop()` to pick the last element from your array (instead of `.reverse()[0]`).